### PR TITLE
Create navbar

### PR DESCRIPTION
**Pull request by themikail**

### CODE REVIEW

File: pages/components/feed.js [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+    <FeedSection>
       <ActiveMusic />
       <Cards />
       <Navbar />
-    </div>
+    </FeedSection>
   );
 }
+
+const FeedSection = styled.div`
```
Comment (MerleMak) [inline pages/components/feed.js:16]: Why not use a section element if its already named after it?:D

File: pages/index.js [hunk 2]
```diff
@@ -1,9 +1,79 @@
-import Feed from "./components/feed";
+import Head from "next/head";
+import Image from "next/image";
+import Link from "next/link";
+import { Inter } from "next/font/google";
+import styles from "@/styles/Home.module.css";
```
Comment (MerleMak) [inline pages/index.js:5]: We initially agreed on styling the capstone app with only styled components. Mixing css modules and styled components is considered bad practice and unfortunately I have to ask you to rewrite your styling into styled components.

File: pages/index.js [hunk 3]
```diff
@@ -1,9 +1,79 @@
...
   return (
     <>
-      <Feed />
+      <Head>
+        <title>Music</title>
+        <meta name="description" content="Generated by create next app" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <GlobalStyle />
```
Comment (MerleMak) [inline pages/index.js:19]: the GlobalStyle is not correctly implemented here. it goes into the _app.js :)

File: pages/index.js [hunk 4]
```diff
@@ -1,9 +1,79 @@
...
+      <StyledBody>
+        <main className={`${styles.main} ${inter.className}`}>
+          <Title>
+            <h1>Share youre lovely music with Music</h1>
```
Comment (MerleMak) [inline pages/index.js:23]: I think you have to rephrase this. I am not sure what you mean with this sentence :)

File: pages/index.js [hunk 5]
```diff
@@ -1,9 +1,79 @@
...
-      <Feed />
+      <Head>
+        <title>Music</title>
+        <meta name="description" content="Generated by create next app" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <GlobalStyle />
+      <StyledBody>
+        <main className={`${styles.main} ${inter.className}`}>
```
Comment (MerleMak) [inline pages/index.js:21]:
```suggestion
        <StyledMain>
```

File: pages/index.js [hunk 6]
```diff
@@ -1,9 +1,79 @@
...
+            <SpotifyLogo>
+              <Image
+                src="/assets/images/login/spotify/icon4@2x.png"
+                width={45}
+                height={45}
+                alt="Spotify"
+              />
+            </SpotifyLogo>
+          </StyledButton>
+        </main>
```
Comment (MerleMak) [inline pages/index.js:38]:
```suggestion
        </StyledMain>
```

File: styles/Home.module.css [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+}
+
+@keyframes rotate {
+  from {
+    transform: rotate(360deg);
+  }
+  to {
+    transform: rotate(0deg);
+  }
+}
```
Comment (MerleMak) [inline styles/Home.module.css:229]: I am sorry, but you will have to wrap these styles into styled components.. 
This is part of our minimum requirements

**Review by MerleMak — CHANGES_REQUESTED** [review]
Hi Mikail,
You App looks good so far. Its important to not that css modules are not how we style the app. It has to be styled components. Once you refactored your styling you can tag me and I can check it :)

File: pages/index.js (hunk 4 above)
Comment (themikail) [reply pages/index.js:19]: it is a dummy sentence. i will change it in the course of time

**Review by MerleMak — COMMENTED** [review]
Looks better now :)

File: pages/components/feed.js (hunk 1 above)
Comment (themikail) [reply pages/components/feed.js:16]: what do you mean. Did not understand  :O

**Pull request merged**